src/quo/previews/preview.cljs

```diff
@@ -2,7 +2,8 @@
   (:require [reagent.core :as reagent]
```

Comment:
not sure why changes in this file ? it's not used

Reply:
True, I should be using `status-im.ui2.screens.quo2-preview.preview`, I will change it